retrieval_service/datastore/providers/firestore.py

```diff
@@ -76,6 +79,68 @@ async def delete_collections(collection_list: list[AsyncCollectionReference]):
             [airports_ref, amenities_ref, flights_ref, policies_ref]
         )
 
+        async def delete_indexes(index_list):
```

Comment:
nit: we could probably have this as a private function so it is not nested in this function

Reply:
delete collections is also nested, should I change both?

Reply:
yes, that will make this code more readable